test/index.js

```diff
@@ -9,6 +9,7 @@ var grunt = require('grunt'),
 test('Grunt-localizr', function (t) {
```

Comment:
I've submitted a PR to your master branch: https://github.com/pvenkatakrishnan/grunt-localizr/pull/1